pkg/pillar/hypervisor/xen.go

```diff
@@ -102,7 +102,7 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 
 	switch config.VirtualizationMode {
 	case types.PV:
-		xen_type = "pvh"
+		xen_type = "pv"
```

Comment:
How about adding a comment that xen-start will change this to pvh if it is supported?

Reply:
Good call -- will add a generic comment block!